tock/employees/models.py

```diff
@@ -105,6 +103,9 @@ class UserData(models.Model):
     is_18f_employee = models.BooleanField(default=True, verbose_name='Is 18F Employee')
     is_billable = models.BooleanField(default=True, verbose_name="Is 18F Billable Employee")
     unit = models.IntegerField(null=True, choices=UNIT_CHOICES, verbose_name="Select 18F unit", blank=True)
+    billable_expectation = models.DecimalField(validators=[MaxValueValidator(limit_value=1)],
+                                              default=0.80, decimal_places=2, max_digits=3,
+                                              verbose_name="Percentage of hours which are expected to be billable each week")
```

Comment:
Is it a percentage, or a number? I had _thought_ a number?

Reply:
Maybe say "Percentage of hours (expressed as a decimal) expected to be billable"?